config/initializers/locale.rb

```diff
@@ -0,0 +1,14 @@
+# have fallbacks as described here: https://github.com/svenfuchs/i18n/wiki/Fallbacks
+require 'i18n/backend/fallbacks'
+
+# Set default locale to English
+I18n.default_locale = :en
+
+# Whitelist locales available for the application
+I18n.available_locales = %i[espanolamericalatina en es lietuvos 한국어 english espanol فارسی]
+
+# Allow nested diretories in locales
+I18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
+
+# custom TCE fallbacks
+I18n.fallbacks.map(espanolamericalatina: :espanol)
```

Comment:
this is supposed to work according to the i18n gem, but it isn't working for me locally

Reply:
i gave up trying to do fallbacks. think it wasn't working because of the non-standard way i am using I18n in TCE